src/game/default/g_entity.c

```diff
@@ -71,6 +133,9 @@ static g_entity_spawn_t g_entity_spawns[] = { // entity class names -> spawn fun
 
 	{ "worldspawn", G_worldspawn },
 
+	// compatibility-only entities
+	{ "weapon_chaingun", G_weapon_chaingun },
+
```

Comment:
I think we should make like a g_q2_compat.c which exists solely to satisfy compatibility with Quake 2 maps, in the case that we need to do so. That way this stuff isn't just cluttering other Quetoo-related files.